src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -781,9 +781,30 @@ public PreferencesViewModel(DynamoViewModel dynamoViewModel)
 
             PackagePathsViewModel.RootLocations.CollectionChanged += PackagePathsViewModel_RootLocations_CollectionChanged;
 
+            WorkspaceEvents.WorkspaceSettingsChanged += PreferencesViewModel_WorkspaceSettingsChanged;
+
             PropertyChanged += Model_PropertyChanged;
         }
 
+        /// <summary>
+        /// This method will be executed everytime the WorkspaceModel.ScaleFactor value is updated.
+        /// </summary>
+        /// <param name="args"></param>
+        private void PreferencesViewModel_WorkspaceSettingsChanged(WorkspacesSettingsChangedEventArgs args)
+        {
+            //The WorkspaceSettingsChanged event is refering to the double ScaleFactor, then we need to make the conversion to Logarithm scale factor ScaleFactorLog       
+            UpdateGeoScaleRadioButtonSelected(Convert.ToInt32(Math.Log10(args.ScaleFactor)));
+        }
+
+        /// <summary>
+        /// This method will update the currently selected Radio Button in the Geometry Scaling section.
+        /// </summary>
+        /// <param name="scaleFactor"></param>
+        public void UpdateGeoScaleRadioButtonSelected(int scaleFactor)
```

Comment:
seems this can be internal or private.

Reply:
@mjkkirschner this method was changed to private in the next commit:
https://github.com/DynamoDS/Dynamo/pull/11845/commits/f3acc0bfdba9db78dfd2e2bd2ab53e2f098fa7c2